Dockerfile

```diff
@@ -30,6 +34,7 @@ RUN \
 
 COPY --from=builder /app/bin /app/bin
 COPY --from=builder /app/version.json /app
+COPY --from=builder /app/mmdb /app/mmdb
```

Comment:
I guess this now needs to be rebased after https://github.com/mozilla-services/pairsona/pull/25

Reply:
Sort of. Hilariously, i had fixed it here before i did #25.